index.html

```diff
@@ -5,12 +5,96 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&display=swap" rel="stylesheet">
+
+    <title>Adiba's Bakery</title>
+    <link rel="stylesheet" href="/style.css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="container">
+        <header class="header-description">
+            <div class="logo"></div>
+            <div class="flex-nav">
+                <div class="nav-button">
+                    <button class="nav-toggle" id="navToggle">
+                        <i class="fas fa-bars"></i>
+                    </button>
+                </div>
+            </div>
+
+            <div class="navbar-description">
+                <nav class="nav-links" id="nav-links">
+                    <ul>
+                        <li class="nav-link"><a href="#"> HOME</a></li>
+                        <li class="nav-link"><a href="#"> CAKES </a></li>
+                        <li class="nav-link"><a href="#"> ORDERIN</a></li>
+                        <li class="nav-link"><a href="#"> LESSONS</a></li>
+                        <li class="nav-link"><a href="#"> ABOUT</a></li>
+                    </ul>
+                </nav>
+            </div>
+    <div class="h2">
+        <h2> The best cakes in town delevered to your door.
+        </h2>
+    </div>
+
+    </header>
+
+    <section class="first-section">
```

Comment:
you've used a header & footer tag which is great, here you could also use a `main` tag